tests/e2e/vms/xsvm.go

```diff
@@ -0,0 +1,163 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package vms
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"path/filepath"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/avalanchego/config"
+	"github.com/ava-labs/avalanchego/tests"
+	"github.com/ava-labs/avalanchego/tests/fixture/e2e"
+	"github.com/ava-labs/avalanchego/tests/fixture/tmpnet"
+	"github.com/ava-labs/avalanchego/utils/crypto/secp256k1"
+	"github.com/ava-labs/avalanchego/utils/units"
+	"github.com/ava-labs/avalanchego/vms/example/xsvm"
+	"github.com/ava-labs/avalanchego/vms/example/xsvm/cmd/issue/export"
+	"github.com/ava-labs/avalanchego/vms/example/xsvm/cmd/issue/importtx"
+	"github.com/ava-labs/avalanchego/vms/example/xsvm/cmd/issue/transfer"
+	"github.com/ava-labs/avalanchego/vms/example/xsvm/genesis"
+
+	ginkgo "github.com/onsi/ginkgo/v2"
+)
+
+var (
+	subnetAName = "xsvm-a"
+	subnetBName = "xsvm-b"
+)
+
+func XSVMSubnets(nodes ...*tmpnet.Node) []*tmpnet.Subnet {
+	return []*tmpnet.Subnet{
+		newXSVMSubnet(subnetAName, nodes...),
+		newXSVMSubnet(subnetBName, nodes...),
+	}
+}
+
+var _ = ginkgo.Describe("[XSVM]", func() {
+	require := require.New(ginkgo.GinkgoT())
+
+	ginkgo.It("should support transfers between subnets", func() {
+		network := e2e.Env.GetNetwork()
+
+		pluginDir, err := network.DefaultFlags.GetStringVal(config.PluginDirKey)
+		require.NoError(err)
+		const xsvmPluginFilename = "v3m4wPxaHpvGr8qfMeyK6PRW3idZrPHmYcMTt7oXdK47yurVH"
```

Comment:
nit: this string seems duplicated in `vms/example/xsvm/vm.go`. Consider defining a variable in that fila and reuse it here